build/tasks/process.js

```diff
@@ -30,6 +30,12 @@ module.exports = function (grunt) {
       proc.kill('SIGINT');
     } else if (!proc) {
       procs[this.target] = spawn('node', [filePath]);
+
+      if (grunt.verbose) {
+        var logStream = fs.createWriteStream('services.log', {flags: 'a'});
+        procs[this.target].stdout.pipe(logStream);
+        procs[this.target].stderr.pipe(logStream);
+      }
```

Comment:
You should have access to `process.argv`... I have done this before, but I don't see why it would not work.